default.py

```diff
@@ -319,17 +319,21 @@ def ask_bitrate(self, bitrates):
         else:
             return None
 
-    def select_bitrate(self, manifest_bitrates=None):
+    def select_bitrate(self, manifest_bitrates=None, manifest_resolution=None):
         """Returns a bitrate, while honoring the user's /preference/."""
+        manifest_bitrates = manifest_bitrates.keys()
+        print ('Das ist ein Test der Output ist: ', manifest_bitrates)
```

Comment:
Can you remove these lines? Or, if you find them useful, please translate them to english and use the logging interface.

---Alex